<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>s21::View</class>
 <widget class="QMainWindow" name="s21::View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Maze 'n' Cave</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>411</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="maze_tab">
     <attribute name="title">
      <string>Maze</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="cave_tab">
     <attribute name="title">
      <string>Cave</string>
     </attribute>
     <widget class="QPushButton" name="transform_cave_button">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>121</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Transform Cave</string>
      </property>
     </widget>
     <widget class="QPushButton" name="create_cave_button">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Create Cave</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="rows_spinbox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="suffix">
       <string>  rows</string>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="cols_spinbox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="suffix">
       <string>  cols</string>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="death_spinbox">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="suffix">
       <string>  death limit</string>
      </property>
      <property name="maximum">
       <number>7</number>
      </property>
      <property name="value">
       <number>4</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="birth_spinbox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="suffix">
       <string>  birth limit</string>
      </property>
      <property name="maximum">
       <number>7</number>
      </property>
      <property name="value">
       <number>3</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="birth_chance_spinbox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="suffix">
       <string>%, birth chance</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="singleStep">
       <number>5</number>
      </property>
      <property name="value">
       <number>30</number>
      </property>
     </widget>
     <widget class="QPushButton" name="flip_cave_button">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>100</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Flip Cave</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="draw_area">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
